modal/_vendor/cloudpickle.py

```diff
@@ -323,6 +323,9 @@ def _extract_code_globals(co):
 
     return out_names
 
+def _extract_code_attr(co):
+    out_names = {name: None for name in _walk_attr_ops(co)}
+    return out_names
```

Comment:
Hm, I don't think that we should be making changes in vendored files; it gets too confusing if we ever need to update them from the upstream source.

Reply:
Moved everything into function_utils but things may be a little sloppier now (?)